packages/components/src/utils/useMediaQueries.tsx

```diff
@@ -45,7 +44,8 @@ export const useMediaQueries = (
       },
     }
   }
-  const theme = useTheme()
+
+  const breakpoints = { medium: "768px", large: "1080px" }
```

Comment:
Given it's meant to match values in our CSS vars, probably worth leaving a comment here and in the tokens source (if possible)